### PR TITLE
Dashboard Integration

### PR DESCRIPTION
**Pull request by rsandb**

### CODE REVIEW

**Comment by rsandb** [comment]
This one should work

File: website/src/pages/terms-of-service.tsx [hunk 1]
```diff
@@ -1,148 +1,27 @@
-import { Container, Heading } from "@chakra-ui/react";
+import { Box, Heading, Stack } from "@chakra-ui/react";
```
Comment (fozziethebeat) [inline website/src/pages/terms-of-service.tsx:1]: Could you pull out the terms of service changes since that's in a separate PR?

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
Just one set of simplifications needed then I'll approve.

File: website/src/pages/terms-of-service.tsx (hunk 1 above)
Comment (rsandb) [reply website/src/pages/terms-of-service.tsx]: Of course, sorry I thought I took them out.

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**